### PR TITLE
Dockerize project

### PR DESCRIPTION
**Pull request by agviegas**
Watch out the changes I did to package.json (ignoring ifc2x4.ts and using ifc2x4.js instead). There seems to be something wrong with ifc2x4.ts. This is a temporary fix, but I'd try to clone the porject on your end and see where the problem is.

### CODE REVIEW

**Comment by tomvandig** [comment]
I've committed ifc2x4.ts, so you can use that directly now.

File: package.json [hunk 1]
```diff
@@ -26,18 +26,19 @@
     "release_publish": "npm run build-release && cd dist && npm publish",
     "build-wasm-debug": "em++ --bind -O3 -gsource-map -std=c++17 --source-map-base http://localhost:5000/web-ifc-js/wasm-lib/ -flto -fno-exceptions ./src/wasm/web-ifc-api.cpp -s ALLOW_MEMORY_GROWTH=1 -s MAXIMUM_MEMORY=4GB -s ASSERTIONS=1 -s FORCE_FILESYSTEM=1 -s EXPORT_NAME=WebIFCWasm -s MODULARIZE=1 -s EXPORTED_RUNTIME_METHODS=[\"FS\"] -O3 -o dist/web-ifc.js",
     "build-wasm-release": "em++ --bind -O3 -std=c++17 -flto -fno-exceptions ./src/wasm/web-ifc-api.cpp -s ALLOW_MEMORY_GROWTH=1 -s MAXIMUM_MEMORY=4GB -s FORCE_FILESYSTEM=1 -s EXPORT_NAME=WebIFCWasm -s MODULARIZE=1 -s EXPORTED_RUNTIME_METHODS=[\"FS\"] -O3 -o dist/web-ifc.js",
-    "build-api": "cpy src/*.ts dist && npm run build-ts-api && npm run build-web-ifc-api-mjs && npm run build-web-ifc-api-node && npm run copy-to-dist && npm run add-wasm-path",
+    "build-api": "cpy src/*.ts dist && cpy src/ifc2x4.js dist && npm run build-ts-api && npm run build-web-ifc-api-mjs && npm run build-web-ifc-api-node && npm run copy-to-dist && npm run add-wasm-path",
```
Comment (tomvandig) [inline package.json:29]: This should be unnecessary now

**Review by tomvandig — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (tomvandig) [reply package.json:29]: ifc2x4.ts is in the src/ folder now

Comment (agviegas) [reply package.json:29]: Done!

File: package.json [hunk 2]
```diff
@@ -30,13 +30,15 @@
     "build-ts-api": "tsc --emitDeclarationOnly && cpy dist/web-ifc-api.d.ts dist && cpy dist/web-ifc-api.d.ts dist --rename=web-ifc-api-node.d.ts",
     "build-web-ifc-api-mjs": "esbuild dist/web-ifc-api.ts --bundle --format=esm --external:path --external:fs --outfile=./dist/web-ifc-api.js",
     "build-web-ifc-api-node": "esbuild dist/web-ifc-api.ts --bundle --platform=node --outfile=./dist/web-ifc-api-node.js",
-    "copy-to-dist": "cpy README.md dist && cpy package.json dist && cpy src/ifc2x4.ts dist",
+    "copy-to-dist": "cpy README.md dist && cpy package.json dist && cpy src/ifc2x4.js dist",
```
Comment (tomvandig) [inline package.json:33]: Is this still needed?

**Review by tomvandig — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (agviegas) [reply package.json:33]: Fixed!

Comment (tomvandig) [reply package.json:33]: I'm still seeing it. Am I missing something? @agviegas

Comment (agviegas) [reply package.json]: Woops, my bad. I didn't push.

Comment (agviegas) [reply package.json]: Ready!

**Pull request merged**